livekit-plugins/livekit-plugins-assemblyai/livekit/plugins/assemblyai/stt.py

```diff
@@ -0,0 +1,423 @@
+# Copyright 2023 LiveKit, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from __future__ import annotations
+
+import asyncio
+import dataclasses
+import json
+import os
+import time
+from contextlib import suppress
+from dataclasses import dataclass
+from typing import List, Optional, Union, Literal
+from urllib.parse import urlencode
+
+import aiohttp
+from livekit import rtc
+from livekit.agents import stt, utils
+from livekit.agents.utils import AudioBuffer
+
+
+ENGLISH = "en"
+
+# Define bytes per frame for different encoding types
+bytes_per_frame = {
+    "pcm_s16le": 2,
+    "pcm_mulaw": 1,
+}
+
+
+@dataclass
+class STTOptions:
+    sample_rate: Optional[int] = None
+    word_boost: Optional[List[str]] = None
+    encoding: Optional[Literal["pcm_s16le", "pcm_mulaw"]] = None
+    disable_partial_transcripts: bool = False
+    enable_extra_session_information: bool = False
+    end_utterance_silence_threshold: Optional[int] = None
+    buffer_size_seconds: Optional[float] = None
+    token_expires_in: Optional[int] = None
+
+
+class STT(stt.STT):
+    def __init__(
+        self,
+        *,
+        api_key: Optional[str] = None,
+        sample_rate: Optional[int] = 16000,
+        word_boost: Optional[List[str]] = None,
+        encoding: Optional[str] = "pcm_s16le",
+        disable_partial_transcripts: bool = False,
+        enable_extra_session_information: bool = False,
+        end_utterance_silence_threshold: int = 1000,
+        http_session: Optional[aiohttp.ClientSession] = None,
+        token_expires_in: int = 3600,
+        buffer_size_seconds: float = 0.2,
+    ):
+        super().__init__(streaming_supported=True)
```

Comment:
This parameter is no longer working with the newest versions. I assume the interface was changed.
This worked for me:
```python
capabilities = stt.STTCapabilities(streaming=True, interim_results=True)
super().__init__(capabilities=capabilities)
```